src/blocker.rs

```diff
@@ -2,18 +2,17 @@
 
 use memchr::{memchr as find_char, memrchr as find_char_reverse};
 use once_cell::sync::Lazy;
-use serde::{Deserialize, Serialize};
-use std::collections::{HashMap, HashSet};
+use serde::Serialize;
+use std::collections::HashSet;
 use std::ops::DerefMut;
-use std::sync::Arc;
 use thiserror::Error;
 
-use crate::filters::network::{NetworkFilter, NetworkFilterMaskHelper, NetworkMatchable};
-use crate::optimizer;
+use crate::filters::network::{NetworkFilter, NetworkFilterMaskHelper};
+pub(crate) use crate::network_filter_list::NetworkFilterList;
```

Comment:
let's drop the `pub(crate)` here, and change other imports to refer to it directly from `crate::network_filter_list`